winml/test/common/testPch.h

```diff
@@ -10,3 +10,8 @@
 #include <wrl/implements.h>
 
 #include "fileHelpers.h"
+// Windows pollutes with preprocessor that redefine OPTIONAL.
+// Undefine OPTIONAL to get onnx macros to resolve correctly.
+#ifdef OPTIONAL
+#undef OPTIONAL
+#endif
```

Comment:
This was copied from the concurrency test's PCH

Reply:
Should this be added after windows.h, or are we not sure where the OPTIONAL define is coming from?